azurerm/resource_arm_dns_ns_record.go

```diff
@@ -33,9 +35,21 @@ func resourceArmDnsNsRecord() *schema.Resource {
 				Required: true,
 			},
 
+			"records": {
```

Comment:
can we also update the documentation for this field? this is hosted in the `website/docs/r/dns_ns_record.html.markdown` file

Reply:
Done. I added a warning that one or the other is required, but I'm open to changing the verbiage.